messaging/src/main/java/io/lantern/messaging/metadata/Metadata.kt

```diff
@@ -25,7 +25,7 @@ private val logger = KotlinLogging.logger {}
 /**
  * Provides a facility for extracting content metadata while copying it
  */
-class Metadata(val mimeType: String?, val thumbnail: ByteArray?, val thumbnailMimeType: String?) {
+class Metadata(val mimeType: String?, val duration: Double?, val thumbnail: ByteArray?, val thumbnailMimeType: String?) {
```

Comment:
With this now we can fetch the duration directly from the metadata

Reply:
Duration is specific to audio files. Metadata is supposed to be a more general class, which is why I suggested making a Map of `additionalMetadata`.

Reply:
Also, the duration needs to actually make it into the attachment, which this PR doesn't do.